CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 7.0.0 - 2019/06/03
+
+### Breaking changes
+
+* Starting with ansible-elasticsearch:7.0.0, installing more than one instance of Elasticsearch **on the same host** is no more supported.
+* Configuration, datas, logs and PID directories are now using standard paths like in the official elasticsearch packages.
+
+* If you use only one instance but want to upgrade from an older ansible-elasticsearch version, follow [upgrade procedure](./docs/multi-instance.md#upgrade-procedure)
+* If you install more than one instance of ElasticSearch on the same host (with different ports, directory and config files), **do not update to ansible-elasticsearch >= 7.0.0**, please follow this [workaround](./docs/multi-instance.md#workaround) instead.
```

Comment:
ElasticSearch/Elasticsearch. I think this was the old way of writing it, and I know it annoys some people so best to be consistent. I also see a few "elasticsearch" in this changelog too.